ext/mbstring/libmbfl/mbfl/mbfl_encoding.c

```diff
@@ -35,10 +35,6 @@
 #include <stddef.h>
 #include <string.h>
 
-#ifdef HAVE_STRINGS_H
-#include <strings.h>
-#endif
```

Comment:
This file uses `strcasecmp()`, which is defined by strings.h according to POSIX. Probably it works without it because string.h likely reexports it for historical reasons.

Reply:
Thank you for this... Got it. Should we include strings.h file conditionally like now? Because in that case I'll just move one check from the ext/date/config.m4 to configure.ac instead... Probably other strings.h usages should follow the same pattern. These POSIX headers are otherwise probably not problematic on most platforms since they are very common. But would be good to check this a bit.

Reply:
I'm not totally sure, but I think Windows doesn't have strings.h, so conditional include is probably necessary.

Reply:
Indeed, the WinSDK doesn't have strings.h.

Reply:
Thanks for the info. Worth noting that the strcasecmp() function is similarly used all across the php. Is it possible that it is basically relying on the strings.h file that is included from the ext/date/lib or the one from the <string.h> C89 compliant one that accidentally includes also this for the compatibility sake on most platforms... I'll check it out and either adjust this pr or close it. Well, if it works, fixing something that isn't broken of course isn't good then. :)

Thank you for your feedback.